test/test_core.py

```diff
@@ -7611,15 +7611,15 @@ def test_embind_val_coro_propogate_cpp_exception(self):
     self.cflags += ['-std=c++20', '--bind', '--pre-js=pre.js', '-fexceptions', '-sINCOMING_MODULE_JS_API=[onRuntimeInitialized]', '--no-entry']
     self.do_runf('embind/test_val_coro.cpp', 'rejected with: std::runtime_error: bang from throwingCoro!\n')
 
-  def test_embind_val_coro_propogate_js_error(self):
+  def test_embind_val_coro_propagate_js_error(self):
     self.set_setting('EXCEPTION_STACK_TRACES')
     create_file('pre.js', r'''Module.onRuntimeInitialized = () => {
       Module.failingPromise().then(
         console.log,
         err => console.error(`rejected with: ${err.message}`)
       );
     }''')
-    self.cflags += ['-std=c++20', '--bind', '--pre-js=pre.js', '-fexceptions']
+    self.cflags += ['-std=c++20', '--bind', '--pre-js=pre.js', '-fexceptions', '-sINCOMING_MODULE_JS_API=[onRuntimeInitialized]', '--no-entry']
```

Comment:
No need for the square backets here

Reply:
(20 occurrences of `-sINCOMING_MODULE_JS_API=[onRuntimeInitialized]`, one occurrence of `-sINCOMING_MODULE_JS_API=arguments` in test_esm_integration.